backend/btrixcloud/users.py

```diff
@@ -328,6 +328,17 @@ async def format_invite(self, invite):
         if invite.oid:
             org = await self.org_ops.get_org_for_user_by_id(invite.oid, inviter)
             result["orgName"] = org.name
+            result["orgSlug"] = org.slug
+
+            result["firstOrgAdmin"] = False
+            org_owners = await self.org_ops.get_org_owners(org)
+            if not org_owners:
+                result["firstOrgAdmin"] = True
+
+            # Require rename in frontend if org name is set to id
+            result["orgNameRequired"] = False
```

Comment:
Should we enforce this on the backend, eg. that the orgName / orgSlug are not optional in this case?